kubernetes/base/deployments/smtp-server.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
```

Comment:
Is it necessary to have SMTP pod as mailing functionality was disabled in demoshop?